site/tutorials/snippets/17/monitoring/setupPrometheus.md

```diff
@@ -0,0 +1,73 @@
+
+## Setup Prometheus Monitoring
+Duration: 3:00
+
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
+
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite. 
+
+* To install the Prometheus and Alert Manager, execute:
+
+    <!-- command -->
+    ```
+    kubectl create namespace monitoring
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm install prometheus prometheus-community/prometheus --namespace monitoring --wait
+    ```
+
+
+### Execute the following steps to install prometheus-service
+
+* Install prometheus-service using `helm`:
+<!-- command -->
+```
+helm upgrade --install -n keptn prometheus-service https://github.com/keptn-contrib/prometheus-service/releases/download/0.8.4/prometheus-service-0.8.4.tgz --reuse-values
```

Comment:
I would prefer to add `--wait` to the help upgrade to ensure that the deployment is ready